osu.Game.Rulesets.Osu/Mods/OsuModMagnetised.cs

```diff
@@ -16,19 +16,19 @@
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    internal class OsuModAimAssist : Mod, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>
+    internal class OsuModMagnetised : Mod, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>
     {
-        public override string Name => "Aim Assist";
-        public override string Acronym => "AA";
-        public override IconUsage? Icon => FontAwesome.Solid.MousePointer;
+        public override string Name => "Magnetised";
+        public override string Acronym => "MG";
+        public override IconUsage? Icon => FontAwesome.Solid.Magnet;
         public override ModType Type => ModType.Fun;
-        public override string Description => "No need to chase the circle – the circle chases you!";
+        public override string Description => "No need to chase the circles – your cursor is a magnet!";
         public override double ScoreMultiplier => 1;
         public override Type[] IncompatibleMods => new[] { typeof(OsuModAutopilot), typeof(OsuModWiggle), typeof(OsuModTransform), typeof(ModAutoplay), typeof(OsuModRelax) };
 
         private IFrameStableClock gameplayClock;
 
-        [SettingSource("Assist strength", "How much this mod will assist you.", 0)]
+        [SettingSource("Attraction strength", "How strong the pull is.", 0)]
         public BindableFloat AssistStrength { get; } = new BindableFloat(0.5f)
```

Comment:
I didn't rename the property here to avoid breaking compatibility, but then realised the mod name is changing so that's a non issue. Think we should rename this to `AttractionStrength` while here, or not important?

Reply:
Sounds good